aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/TagHelpers/AddressScriptTagHelperComponent.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace RazorPagesSample.TagHelpers
+{
+    public class AddressScriptTagHelperComponent : ITagHelperComponent
```

Comment:
For the sake of simplicity I'd inherit from [`TagHelperComponent`](https://github.com/aspnet/Razor/blob/b20921741ced5c647e2e2f810f89613a8d4f7287/src/Microsoft.AspNetCore.Razor/TagHelpers/TagHelperComponent.cs) instead of `ITagHelperComponent` since it implements some of the boiler plate.

Ditto with the other TagHelperComponent as well.